airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/class_types_registry.py

```diff
@@ -56,7 +56,7 @@
 """
 CLASS_TYPES_REGISTRY contains a mapping of developer-friendly string -> class to abstract the specific class referred to
 """
-CLASS_TYPES_REGISTRY: Mapping[str, Type] = {
+CLASS_TYPES_REGISTRY: Mapping[str, type] = {
```

Comment:
why this change?

Reply:
When I make changes to this file, `mypy` finds the following error:
```
airbyte_cdk/sources/declarative/parsers/class_types_registry.py:59: error: Missing type parameters for generic type "Type"  [type-arg]
```

I've created a separate PR to fix the `SessionTokenAuthorization`, and you can see there that `mypy` is failing with this error: https://github.com/airbytehq/airbyte/actions/runs/6719622494/job/18261600772?pr=32040

It seems that when using a `Type` (imported from the `typing`), we also need to specify a parameter in the annotation (`Type[<param>]`). 

Ours doesn't specify a parameter, so I replaced it with a generic `type` to get around the `mypy` error.